WordPress/src/main/res/values/strings.xml

```diff
@@ -2300,6 +2300,7 @@
     <string name="reader_toast_err_unable_to_unfollow_blog">Unable to unsubscribe from blog</string>
     <string name="reader_toast_posts_from_this_blog_blocked">Posts from this blog will no longer be shown</string>
     <string name="reader_toast_err_unable_to_block_blog">Unable to block this blog</string>
+    <string name="reader_toast_success_UNDO">This blog has been unblocked!\nPosts from this blog will still be shown</string>
```

Comment:
This is a minor nit, but the uppercase "UNDO" isn't standard. Can we stick with lowercase?

Also, let's shorten the message to just "Blog unblocked" without punctuation, which is more in line with the [Material writing guidelines](https://m2.material.io/design/communication/writing.html#principles).

Reply:
Works for me! 
I'll change the string to lowercase and shorten the message to "Blog unblocked".
Thanks for the advice!

Reply:
![c2c6ced610f353558e0d748b87da8c7](https://github.com/user-attachments/assets/237a7c13-bbce-4a69-92db-b03f0ae116e2)